analyzer/pkg/dpkg/dpkg.go

```diff
@@ -105,14 +106,16 @@ func (a debianPkgAnalyzer) parseDpkgPkg(scanner *bufio.Scanner) (pkg *types.Pack
 			}
 		} else if strings.HasPrefix(line, "Version: ") {
 			version = strings.TrimPrefix(line, "Version: ")
+		} else if strings.HasPrefix(line, "Status: ") {
+			isInstalled = strings.HasPrefix(strings.TrimPrefix(line, "Status: "), "install")
```

Comment:
Is the order of statuses always the same? I'm worried that the package is not detected when `Status` is like `ok install installed` In that case, we cannot detect any vulnerability of the package. The best way is we'll check the specification. If we can't find it, we should check with Debian 8, 9, 10 and Ubuntu 16.04, 18.04, 20.04.

Reply:
@knqyf263 Updated